kit/update.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"fmt"
+	"os/exec"
+)
+
+func update() error {
+	fmt.Println("Updating leapkit/kit...")
+	cmd := exec.Command("go", "install", "github.com/leapkit/leapkit/kit@latest")
```

Comment:
@MateoCaicedoW this is good. However, sometimes this does not pull the latest. We should find the latest kit tag in the repo and use it to update.

Reply:
@paganotoni Comments applied